backend/scripts/create_db.py

```diff
@@ -10,16 +10,29 @@
 pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))  # noqa
 sys.path.insert(0, pkg_root)  # noqa
 
-from corpora.common.corpora_config import CorporaDbConfig
-from corpora.common.corpora_orm import Base
+from .corpora_config import CorporaDbConfig
+from .corpora_orm import Base
+
+
+from sqlalchemy.schema import DropTable
+from sqlalchemy.ext.compiler import compiles
+from server.db.cellxgene_orm import Base as CellxGeneBase
+
+
+@compiles(DropTable, "postgresql")
```

Comment:
So what all is happening in this function?

Reply:
It appends cascade to the drop table command to handle foreign keys/dependencies